projects/observability/src/shared/components/table/data-cell/entity/entity-table-cell-renderer.component.ts

```diff
@@ -1,15 +1,19 @@
 import { ChangeDetectionStrategy, Component } from '@angular/core';
 import { TableCellAlignmentType, TableCellRenderer, TableCellRendererBase } from '@hypertrace/components';
-import { Entity } from '../../../../graphql/model/schema/entity';
 import { ObservabilityTableCellType } from '../../observability-table-cell-type';
+import { MaybeInactiveEntity } from './entity-table-cell-parser';
 
 @Component({
   selector: 'ht-entity-table-cell-renderer',
   styleUrls: ['./entity-table-cell-renderer.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush,
   template: `
     <div class="fill-container entity-cell" [ngClass]="{ 'first-column': this.isFirstColumn }">
-      <ht-entity-renderer [entity]="this.value" [navigable]="true"></ht-entity-renderer>
+      <ht-entity-renderer
+        [entity]="this.value"
+        [inactive]="this.value?.isInactive === true"
```

Comment:
Why not read this off the value that's already passed in?

Reply:
The `value` isn't actually an `Entity` as the typing would have you believe. Its only the `id` and `type`. I almost changed the `Inputs` to reflect this realization (that threw me off at first too), but thought that would be too much for the PR.

Reply:
an entity is just a minimum of id and type, everything else is optional. I meant that isInactive would be available internally, but that MaybeEntity type shouldn't be expected there so another input is fine.